src/funnels/automation/VaultMinter.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.16;
+
+interface AllocatorVaultLike {
+    function draw(uint256) external;
+    function wipe(uint256) external;
+}
+
+contract VaultMinter {
+    mapping (address => uint256) public wards; // Admins
+    mapping (address => uint256) public buds;  // Whitelisted keepers
+    MinterConfig public config;                // Configuration for keepers
+
+    address public immutable vault; // The address of the vault contract
+
+    struct MinterConfig {
+        int64    num; // The remaining number of times that a mint or burn can be executed by keepers (> 0: mint, < 0: burn)
```

Comment:
```suggestion
        int64    num; // The remaining number of times that a mint or burn can be executed by keepers (> 0: draw, < 0: wipe)
```

Reply:
Fix it and the a few other apparences of mint/burn terminology